src/ZumoKit.ts

```diff
@@ -72,27 +89,46 @@ class ZumoKit {
     return user;
   }
 
+  /**
+   * Fetch historical exchange rates for supported time intervals corresponding to user token set.
+   * On success callback returns historical exchange rates are contained in a mapping between
```

Comment:
Not sure what this is trying to say, reads weird

Reply:
True, this is nonsense. Should have been just `Fetch historical exchange rates for supported time intervals`.